src/com/zombiecastlerush/util/Parser.java

```diff
@@ -0,0 +1,23 @@
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Parser {
+    public static void main(String[] args) throws Exception {
+        new Parser().parse("go North");
```

Comment:
This will not be necessary later as you will not invoke this main method. I would recommend deleting main() method here and then if you want, add unit tests which invoke and ensure the functionality of Parser. Also, since your methods are static, you will be calling Parser.parse, without needing to create a new instance of Parser.

Reply:
Removed main from Parser Class

Reply:
added Unit Test